tests/features.py

```diff
@@ -0,0 +1,41 @@
+from flask import current_app
+from mock import patch
+
+
+def enable_feature(name, value=True):
```

Comment:
Oh, and as David said, this seems to be unused. I say we drop this if that's the case. While there's a possibility that this would be useful, I'd rather not add dead code. Chances are, with no existing usage examples, whoever's going to need this functionality will likely end up implementing it somewhere else anyway.